npm-shrinkwrap.json

```diff
@@ -2825,6 +2820,613 @@
       "from": "fs.realpath@>=1.0.0 <2.0.0",
       "resolved": "https://registry.npmjs.org/fs.realpath/-/fs.realpath-1.0.0.tgz"
     },
+    "fsevents": {
```

Comment:
For shrinkwrap, run the npm script `npm run shinkwrap` which will remove the `fsevents` dependency. `fsevents` fails to install on some machines.